ige/include/ige/plugin/audio/AudioSource.hpp

```diff
@@ -22,9 +22,16 @@ class AudioSource {
     void play();
     void pause();
     void restart();
+    void stop();
 
     bool is_playing();
 
+    void set_looping(bool loop);
```

Comment:
maybe we could default `loop` to `true`? doing just `clip.set_looping()` conveys clearly enough that we want to make it loop